Lesson23/deniz/main.js

```diff
@@ -0,0 +1,14 @@
+const accordions = document.querySelectorAll('.accordion');
+
+accordions.forEach(function (accordion) {
+    accordion.addEventListener('click', function (e) {
+        let accordionBtn = this.querySelector('button');
```

Comment:
This variable doe snot have to be declared with a 'let' keyword as you do not reassign it to anything, you only change value of it's 'textContent' property so it's possible to use 'const' in a case like this. 

No need to change, just an information comment.